logic/src/commonMain/kotlin/com/wire/kalium/logic/data/properties/UserPropertyRepository.kt

```diff
@@ -58,4 +62,23 @@ internal class UserPropertyDataSource(
         userConfigRepository.setReadReceiptsStatus(false)
     }
 
+    override suspend fun getTypingIndicatorStatus(): Boolean =
+        userConfigRepository.isTypingIndicatorEnabled()
+            .firstOrNull()
+            ?.fold({ false }, { it }) ?: false
```

Comment:
question: this indicator is enabled by default and in `userConfigRepository` it returns a `true` by default, also `ObserveTypingIndicatorEnabledUseCase` returns `true` when `Either.Left`, so shouldn't it also return `true` if it's null here as well?

Reply:
You are right, I probably didn't check it when copying the logic from the read receipt feature (which is opt-in). 
As usual great eyes 🐛 🦅